.claude/scripts/pm/validate.sh

```
@@ -0,0 +1,88 @@
+    done
+  fi
+done
+
+[ $warnings -eq 0 ] && [ $errors -eq 0 ] && echo "  ✅ All references valid"
+
+# Check frontmatter
+echo ""
+echo "📝 Frontmatter Validation:"
+invalid=0
+
+for file in $(find .claude -name "*.md" -path "*/epics/*" -o -path "*/prds/*" 2>/dev/null); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)